Dockerfile

```diff
@@ -1,14 +1,121 @@
 ARG UBUNTU_BUILDER=ubuntu:18.04
```

Comment:
Please use `Ubuntu 20.04` as base image

Reply:
ok sounds good

Reply:
@GeekArthur can we update this in blockchain-node as well? https://github.com/helium/blockchain-node/blob/master/.buildkite/Dockerfile-ubuntu-erlang#L1

Assuming your preference is ubuntu 20 now, we're happy to not publish ubuntu 18 artifacts.

Reply:
@abhay Yep, we can update this.